### ISSUE TITLE
Components: FilePicker isn't accessible

### ISSUE THREAD

**Issue by diegohaz** [Automattic/wp-calypso#34729]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/me
2. Try to navigate to upload avatar using keyboard

#### What I expected

The `FilePicker` (`.file-picker`) component should be reachable by keyboard.

#### What happened instead

Focus skips this element and goes straight to the info popover button (which is also not accessbile - #23120).

#### Browser / OS version

Chrome 75 / MacOS Mojave

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

As there has been no recent activity on this issue it is being closed with the label `Closed - Won't Fix`.
If this issue should remain open, please feel free to re-open this bug.
